earth2mip/schema.py

```diff
@@ -349,7 +350,8 @@ class EnsembleRun(pydantic.BaseModel):
     simulation_length: int
     # TODO make perturbation_strategy an Enum (see ChannelSet)
     perturbation_strategy: PerturbationStrategy = PerturbationStrategy.correlated
-    single_value_perturbation: bool = True
+    ic_perturbed_channels: Union[str, List[str]] = 'all'
```

Comment:
try to avoid abbreviations, maybe don't need "ic" at all. . The type hint is also a bit complex. Also, suggest making this Optional, with the default behavior being all channels
```suggestion
    perturbed_channels: Optional[str] = None
```

Reply:
@nbren12 
1. the `[str, List[str]] ` is needed so that one can perturb more than a single channel or `all`
2. Do you prefer `perturbed_channels =None` to signify perturbing all channels, that is a bit ambiguous, no?

Reply:
What is the ambiguity?

There are three possible configurations:
- perturb no channels. this is the same as no perturbation. so can be achieved by perturbation_strategy=None (we should implement that behavior if it isn't there yet).
- perturb all channels. This will be the default, which makes sense if a user has explicitly requested to perturb channels.
- perturb some channels. Specify some channels using `perturbed_channels`.



There are several reasons why I want to avoid things like: `Union[str, List[str]]`.

First, `[str, List[str]] `seems odd to me because only "all" is the only instance of str allowed. The fact that "all" is hardcoded too places is a code smell. `Union[Literal['all'], List[str]]` would be more precise, but in practice, since "all" is the default it is unlikely anyone will every explicitly specifiy it in the config. In this case `Optional[List[str]]=None` will have the same usage and not require a hardcoded string "all" in two separate parts of the code.

Another reason not to use `Union[str, List[str]]` is its best to avoid an overly dynamic config... For example, certain static language like go will not even be able to parse a yaml/json containing this data. OTOH most languages support the idea of None.